docker/types/services.py

```diff
@@ -10,19 +11,15 @@ class TaskTemplate(dict):
 
     Args:
 
-    * container_spec (dict): Container settings for containers started as part
-      of this task. See the :py:class:`~docker.types.services.ContainerSpec`
-      for details.
-    * log_driver (dict): Log configuration for containers created as part of
-      the service. See the :py:class:`~docker.types.services.DriverConfig`
-      class for details.
-    * resources (dict): Resource requirements which apply to each individual
-      container created as part of the service. See the
-      :py:class:`~docker.types.services.Resources` class for details.
-    * restart_policy (dict): Specification for the restart policy which applies
-      to containers created as part of this service. See the
-      :py:class:`~docker.types.services.RestartPolicy` class for details.
-    * placement (list): A list of constraints.
+        container_spec (ContainerSpec): Container settings for containers
```

Comment:
A tip, but not important: these types can be links, like `` :py:class:`~docker.types.services.ContainerSpec` ``.